build/travis/job2_AppImage/build.sh

```diff
@@ -27,17 +29,31 @@ else
   makefile_overrides="" # use Makefile defaults
 fi
 
-# Build MuseScore AppImage inside Docker image
-docker run -i -v "${PWD}:/MuseScore" library/centos:6 \
-   /bin/bash -c "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+# Build AppImage. Are we cross-compiling?
+case "$1" in
+  --32bit )
+    shift
+    # Build MuseScore AppImage inside 32-bit Docker image
+    docker run -i -v "${PWD}:/MuseScore" toopher/centos-i386:centos6 /bin/bash -c \
+      "linux32 --32bit i386 /MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+  * )
+    [ "$1" == "--64bit" ] && shift || true
+    # Build MuseScore AppImage inside native (64-bit) Docker image
+    docker run -i -v "${PWD}:/MuseScore" library/centos:6 /bin/bash -c \
+      "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+esac
```

Comment:
excellent!  I am going to get back on that right with I finish with this current crash issue.  

I was at the point where I was going to use these debian wheezy docker images (https://github.com/multiarch/debian-debootstrap) instead of the raspbian wheezy for a few reasons: (1) Raspbian image is unecesarily larger (2) Raspbian might contain some specific RPi stuff,  (3)  I discovered that I no longer see a raspbian wheezy image on raspberry website, and (4)  I want to consider possibly doing armv7 and armv8 (which is 64-bit) in addition to armv6 (which is the only architecture that raspbian is compiled for) and (5) incase there are any other random architectures now in the future, plain debian will support them all.